.yamato/_projectcontext.yml

```diff
@@ -420,8 +420,6 @@ all_package_ci_project_2019.4:
       -  npm install upm-ci-utils@stable -g --registry https://artifactory.prd.cds.internal.unity3d.com/artifactory/api/npm/upm-npm
       -  upm-ci package izon -t
       -  upm-ci package izon -d
-    triggers:
```

Comment:
Is this one supposed to go?

Reply:
https://unity.slack.com/archives/G0155CW8683/p1623758613025500

Reply:
Good catch, I don't think so, will add it back now

Reply:
Replaced it with general checks triggers, since this runs both package and smoke tests.